tests/foreman/api/test_bookmarks.py

```diff
@@ -367,8 +367,8 @@ def test_negative_update_empty_query(controller):
     assert bm.query != ''
 
 
-@pytest.mark.tier1
-@pytest.mark.parametrize('public', (True, False))
+@pytest.mark.tier1()
+@pytest.mark.parametrize('public', [True, False])
```

Comment:
This is interesting, any particular reason for the change here

Reply:
https://docs.astral.sh/ruff/rules/pytest-parametrize-values-wrong-type